build.gradle

```diff
@@ -33,3 +32,10 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+task gitSubmodulesUpdate {
+    exec {
+        commandLine 'git', 'submodule', 'update', '--recursive'
```

Comment:
Please add instructions under the README to indicate that the clean task is necessary to kick off this command.